bigtop-packages/src/deb/ranger/control

```diff
@@ -35,7 +35,7 @@ Description: Apache Ranger is a framework to enable,
 
 Package: ranger-admin
 Architecture: all
-Depends: adduser, postgresql (>= 8.1), bigtop-utils (>= 0.7), python | python2
+Depends: adduser, postgresql (>= 8.1), libpostgresql-jdbc-java, bigtop-utils (>= 0.7), python | python2
```

Comment:
After applying this PR and #1199, running smoke test on Rocky 8 fails as follows. Would you add postgresql-jdbc to ranger.spec as a dependency just like the deb package?

```
$ ./docker-hadoop.sh -d -dcp -C config_rockylinux-8.yaml -F docker-compose-cgroupv2.yml -G -L -k hdfs,hbase,ranger -s ranger -c 1

...

Notice: /Stage[main]/Ranger::Admin/Package[ranger-admin]/ensure: created
Notice: /Stage[main]/Ranger::Admin/Package[python36]/ensure: created
Notice: /Stage[main]/Ranger::Admin/Exec[initdb]/returns: executed successfully
Notice: /Stage[main]/Ranger::Admin/Service[postgresql]/ensure: ensure changed 'stopped' to 'running'
Notice: /Stage[main]/Ranger::Admin/Exec[change_postgres_password]/returns: executed successfully
Notice: /Stage[main]/Ranger::Admin/File[/usr/lib/ranger-admin/install.properties]/content: content changed '{md5}631fa44e1819ff0a7e64596b37898011' to '{md5}7052aeeb8d8a81c35d282d7a8d76a5f9'
Notice: /Stage[main]/Ranger::Admin/Exec[/usr/lib/ranger-admin/setup.sh]/returns: 2023-10-30 00:01:05,938   --------- Running Ranger PolicyManager Web Application Install Script ---------
Notice: /Stage[main]/Ranger::Admin/Exec[/usr/lib/ranger-admin/setup.sh]/returns: 2023-10-30 00:01:05,940  [I] uname=Linux
Notice: /Stage[main]/Ranger::Admin/Exec[/usr/lib/ranger-admin/setup.sh]/returns: 2023-10-30 00:01:05,942  [I] hostname=fed3e9010829
Notice: /Stage[main]/Ranger::Admin/Exec[/usr/lib/ranger-admin/setup.sh]/returns: 2023-10-30 00:01:05,947  [I] DB_FLAVOR=POSTGRES
Notice: /Stage[main]/Ranger::Admin/Exec[/usr/lib/ranger-admin/setup.sh]/returns: 2023-10-30 00:01:05,950  [I] Audit source=
Notice: /Stage[main]/Ranger::Admin/Exec[/usr/lib/ranger-admin/setup.sh]/returns: 2023-10-30 00:01:05,953  [I] Checking distribution name..
Notice: /Stage[main]/Ranger::Admin/Exec[/usr/lib/ranger-admin/setup.sh]/returns: 2023-10-30 00:01:05,962  [I] Found distribution : Rocky
Notice: /Stage[main]/Ranger::Admin/Exec[/usr/lib/ranger-admin/setup.sh]/returns: 2023-10-30 00:01:05,963  [I] check if command /usr/lib/jvm/java-1.8.0/bin/java exists
Notice: /Stage[main]/Ranger::Admin/Exec[/usr/lib/ranger-admin/setup.sh]/returns: 2023-10-30 00:01:05,965  [I] '/usr/lib/jvm/java-1.8.0/bin/java' command found
Notice: /Stage[main]/Ranger::Admin/Exec[/usr/lib/ranger-admin/setup.sh]/returns: 2023-10-30 00:01:06,026  [I] Checking POSTGRES CONNECTOR FILE : /usr/share/java/postgresql-jdbc.jar
Notice: /Stage[main]/Ranger::Admin/Exec[/usr/lib/ranger-admin/setup.sh]/returns: 2023-10-30 00:01:06,028  [E] POSTGRES CONNECTOR FILE : /usr/share/java/postgresql-jdbc.jar does not exists
Error: '/usr/lib/ranger-admin/setup.sh' returned 1 instead of one of [0]
Error: /Stage[main]/Ranger::Admin/Exec[/usr/lib/ranger-admin/setup.sh]/returns: change from 'notrun' to ['0'] failed: '/usr/lib/ranger-admin/setup.sh' returned 1 instead of one of [0]
Notice: /Stage[main]/Ranger::Admin/Exec[/usr/lib/ranger-admin/set_globals.sh]: Dependency Exec[/usr/lib/ranger-admin/setup.sh] has failures: true
Warning: /Stage[main]/Ranger::Admin/Exec[/usr/lib/ranger-admin/set_globals.sh]: Skipping because of failed dependencies
Warning: /Stage[main]/Ranger::Admin/Exec[systemctl daemon-reload]: Skipping because of failed dependencies
Warning: /Stage[main]/Ranger::Admin/Service[ranger-admin]: Skipping because of failed dependencies
```